src/deck-components/Panel.tsx

```diff
@@ -2,6 +2,11 @@ import { FC, ReactNode } from 'react';
 
 import { findModuleChild } from '../webpack';
 
+export const Panel: FC<{ children?: ReactNode; }> = findModuleChild((mod) => {
```

Comment:
this filter should probably be a bit more specific?

Reply:
It works very reliably in my experience, I think its fine